src/_ffab-active.scss

```diff
@@ -0,0 +1,25 @@
+// - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+// Hover / Active State (ffab)
+// - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+
+$ffab-has-active-state: true !default;
+$ffab-active-transition-speed: 200ms !default;
+$ffab-active-value: 3px !default;
+
+@mixin ffab-active {
+  @if $ffab-has-active-state {
+    .ffab:not(.disabled):not(.no-hover-active) {
```

Comment:
since you have separated the hover and active effects, do you still want to use this class name?  or does it disable the active state as well as the hover effect?